src/styles/_utils.scss

```diff
@@ -1,3 +1,4 @@
+@import "utils/reset";
 @import "utils/vars";
 @import "utils/mixins";
-@import "utils/extends";
```

Comment:
You've deleted the 'extends' import without any explanation. If it is not in use, you can remove it, however, make sure it doesn't affect any other parts of your code.